test/onnx/test_operators.py

```diff
@@ -814,6 +814,10 @@ def test_pixel_shuffle(self):
         x = torch.randn(2, 8, 3, 4).float()
         self.assertONNX(lambda x: torch.pixel_shuffle(x, upscale_factor=2), x, opset_version=11)
 
+    def test_pixel_unshuffle(self):
```

Comment:
This test is failing, every time you add tests to test_operators run python test/onnx/test_operators.py with --accept parameter. See below for more details
```
Usage: python test/onnx/test_operators.py [--no-onnx] [--produce-onnx-test-data]
          --no-onnx: no onnx python dependence
          --produce-onnx-test-data: generate onnx test data
          --accept: accept onnx updates and overwrite models
```